src/libcrun/terminal.c

```diff
@@ -150,7 +150,7 @@ libcrun_terminal_setup_size (int fd, unsigned short rows, unsigned short cols, l
 
   if (ws.ws_row == 0 || ws.ws_col == 0)
     {
-      ret = ioctl (0, TIOCGWINSZ, &ws);
+      ret = ioctl (fd, TIOCGWINSZ, &ws);
```

Comment:
The old code is the desired behavior

If `row == 0` and `col == 0` then it reads the size from the current terminal (at `fd=0`)

Reply:
fine, I now get the code's meaning.